src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,52 @@
+"""Convert SSSOM to ROBOT template."""
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, Union
+
+import pandas as pd
+from sssom import MappingSetDataFrame
+from sssom.parsers import parse_sssom_table
+
+ROBOT_ROW = {
+    'subject_id': 'ID',
+    'subject_label': '',
+    'object_id': 'A oboInOwl:hasDbXref',
+    'object_label': '',
```

Comment:
Seems like columns starting with `>A` (axiom annotations) have to be directly following the main column for this to work `object_id`. Need to drop, or better move, `object_label` column somewhere later.

Reply:
Done!

Reply:
me too tbh